rules/main.smk

```diff
@@ -183,7 +183,7 @@ rule filter_sequences_by_subtype:
     output:
         sequences = "results/{subtype}/{segment}/sequences.fasta",
     params:
-        subtypes=lambda w: config['subtype_query'][w.subtype],
+        subtypes = resolve_subtype_query(),
```

Comment:
_non-blocking_

I have a slight preference for your [original plan](https://github.com/nextstrain/avian-flu/pull/104#discussion_r1982389324) of calling `resolve_config_value` directly instead of creating another wrapper. I think it can set an example for how to access config values that don't require all of the wlidcards.

Reply:
Fair enough - I am ambivalent. The usages of `resolve_config_value` with manually set wildcards are rare (there's only one other case of it in the avian-flu workflows) so I see it as an escape hatch rather than a recommended usage. 

[7205647](https://github.com/nextstrain/avian-flu/pull/146/commits/720564780cdad4709cd904a44bb922ec98b7563a) switches back to the original plan